### PR TITLE
Added examples to tf.clip_by_value

### PR DESCRIPTION
**Pull request by jvishnuvardhan**

### CODE REVIEW

File: tensorflow/python/ops/clip_ops.py [hunk 1]
```diff
@@ -48,6 +48,17 @@ def clip_by_value(t, clip_value_min, clip_value_max,
   Note: `clip_value_min` needs to be smaller or equal to `clip_value_max` for
   correct results.
 
+  For example:
+  
+  ```python
+  A=tf.constant([[1,20,13],[3,21,13]])
```
Comment (alextp) [inline tensorflow/python/ops/clip_ops.py]: Can the code be updated to be compatible with TF style? This means space around = used for assignment, space after commas, etc

File: tensorflow/python/ops/clip_ops.py [hunk 2]
```diff
@@ -48,6 +48,17 @@ def clip_by_value(t, clip_value_min, clip_value_max,
...
+  B=tf.clip_by_value(A, clip_value_min=0, clip_value_max=3) #[[1, 3, 3],[3, 3, 3]]
+  C=tf.clip_by_value(A, clip_value_min=0., clip_value_max=3.)
+  # throws `TypeError` as input and clip_values are of different dtype
```
Comment (alextp) [inline tensorflow/python/ops/clip_ops.py]: The comment on a different line is not obvious; either put it before or just elide this altogether

File: tensorflow/python/ops/clip_ops.py [hunk 3]
```diff
@@ -48,6 +48,17 @@ def clip_by_value(t, clip_value_min, clip_value_max,
...
+  D=tf.constant([2.0,3.1,5.3])
```
Comment (alextp) [inline tensorflow/python/ops/clip_ops.py]: I don't know what value this third example is adding

**Review by alextp — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/clip_ops.py (hunk 3 above)
Comment (jvishnuvardhan) [reply tensorflow/python/ops/clip_ops.py]: Removed third example. Thanks!

File: tensorflow/python/ops/clip_ops.py (hunk 2 above)
Comment (jvishnuvardhan) [reply tensorflow/python/ops/clip_ops.py]: Thanks @alextp. Updated according to your suggestion.

**Review by alextp — DISMISSED** [review]

**Comment by gbaned** [comment]
@jvishnuvardhan Can you please check Ubuntu Sanity errors? Thanks!

**Review by alextp — DISMISSED** [review]

**Review by alextp — APPROVED** [review]

**Pull request merged**